test/library/packages/DistributedBag/add.chpl

```diff
@@ -0,0 +1,20 @@
+/*
+  This test checks the 'distBag.add()' method.
+*/
+
+use DistributedBag;
+
+config param nElems = 3;
+config param defaultMaxSegmentCap = 1024*1024;
```

Comment:
Wouldn't we want this to evolve along with the distributedBagMaxSegmentCap in the module?

For example, you could write `const defaultMaxSegmentCap = DistributedBag.distributedBagMaxSegmentCap`.

Reply:
Exactly